src/main/java/org/ibp/api/rest/study/StudyInstanceResource.java

```diff
@@ -69,38 +72,66 @@
 
 	@ApiOperation(value = "Get study instances with basic metadata.",
```

Comment:
Modify API note (remove plural on "instances") since for `getStudyInstance` since it's only retrieving a single instance